cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1491,6 +1544,15 @@ function start-etcd-servers {
   prepare-etcd-manifest "-events" "4002" "2381" "100m" "etcd-events.manifest"
 }
 
+function start-etcd-grpc-proxy {
+  echo "Start etcd grpc proxy pod..."
+
+  prepare-log-file /var/log/etcd-grpc-proxy.log
+  prepare-etcd-grpc-proxy-manifest "" "2379" "23790" "200m" "etcd-grpc-proxy.manifest"
+
+  ETCD_SERVERS=${ETCD_GRPC_PROXIES:-http://127.0.0.1:23790}
```

Comment:
nit: use a string variable for 23790 here and line 1551

Reply:
Actually, I did not use a variable for "23790" as I found there is no variable for "2379", which the port for core etcd server. I guess the reason is that it is a script, which is hard to debug and thus readability is more important.